src/py21cmfast/_cfg.py

```diff
@@ -16,68 +19,59 @@ class ConfigurationError(Exception):
     pass
 
 
+# TODO: force config to be a singleton
 class Config(dict):
     """Simple over-ride of dict that adds a context manager."""
 
     _defaults = {
-        "direc": "~/21cmFAST-cache",
+        "direc": Path("~/21cmFAST-cache").expanduser(),
         "regenerate": False,
         "write": True,
         "cache_param_sigfigs": 6,
         "cache_redshift_sigfigs": 4,
         "ignore_R_BUBBLE_MAX_error": False,
```

Comment:
I actually don't think any of these settings are being used any more :/ We probably _should_ be using the sigfigs -- we shouldn't lose that functionality. But I don't think the new output struct framework actually accesses this info any more.

Reply:
I can remove these as necessary, but I think it's a good idea to have *some* place we can put variables which don't affect the run output, and may need to be passed in.

Reply:
Yeah, I think having this structure around makes sense. I think for this PR, let's leave everything in there. Once we merge the output struct PR, we'll have a better sense of what is still required.